cert-walletTests/Resources/sample_signed_cert-valid-1.2.0.json

```diff
@@ -1 +1 @@
WDc8xT43DKDpJAKI7nRiG5KDLHCUaMCuPnAE9cj3FwZJgjrHHMlMIJO7Vg02Edzm1TkIHz3JQa4jTj2HzjmGsYDvPG6Kg6MlYDC4MdqrVFySZqYXF2n9N8UbmEBQniUT6H8Sm0wJHfuJYTgGN1pzhWGzXMKCDIUWOIyylHTj1yarSs4cW5dhNkZCkoQyzOLnOOnIoYVMyg4Uk4FokS47Oh4fAmeBHOdXNstawtzAKBwjkqm5/Lvr44cTg0TtQKg2Aok//jiTgWSrGFqB3GEVuOznI9lVo4Q871T04FIPBSLAJBEGTBA716qt7CvIaKEsR8gyCsznHEdqoGVtSBGqq1RVWROFOPB+JoK8eqOTZLgxjIC5HzCi5TdaeoPcaiQBG9OMJMiR4WibEaXnvqJaZu1poHCePPCx9geHiU/Jx3XpcrzqTHPJkTZ6lZJePsNd+NFdfmvDMK5hwztaNY7TjEv2qcoTJDhFFRC7aUsbye8aP+Fh4yCEK5VErzlI2QjAoc5THWCM8oKME1jDtVAwvPwf3yMjdxr7zUDwQJmeKekD3qdOFlQk9TdacgCFVL8hpmOeHLGl7NkANBDgJBouZRXq6GUIzD+6zWZSG3qCJCYQFWYmodUZSAMj1D1enBgAINEDCKixFykYNQSGJsnNLgy/pXqwgC/oxBWJMThDyG1Z3CaPxeSN6QiXAsb2X+sqoGFnV2h2ptlYvTEEHye4UHWVV3ikosZQUaE2TGMlDWkRoKL1AOYVOUiclvF4TK620Ri48RhISYHCEWDFZfqrZwPavk2Dg1BIkK+dyLhBjDgYzkPLFa44EgMiElBfKije2MDe2ATdXAeqWRWlvkbpTwzEMsZIqSqMhB9RHyEXAnpItrp+pOQfIpggzV8JphPst2bdGDlHWkiNtJ7KJaH4YVBZ5Z+cst0Ki3hTdAMdTqJaEcCrGYP14Gw6WeFwkxOQmliyhgxz3HxtmUIBg9IQrlQiEkORFV/SACD0eDIHiAqHTIMyFIws4YMXtekC0sYYgg5BJjdaAoccOOW1lri80FVvucICTphLjULUN2ZIbQhKS5x5yqO0U/Qqwoxo3cpQcpa3ix+DlJH+H6UB0pSIABR06B4RKKRPW+fCj6U7UPUtAoPofHYZcJoy4b247554yNkbClOjXOUA6S18QiRMHgSHIpvBYVxOP+GCS7OBRbLo2cfAJj5PtoJN54x7Js0NgzlKk6UGO1tvCmMQfkL+vvkqDzH/qIulmRG47dj1pUZR0rPBVh71QNr01/FWpZ15GFFFsZeIFByzpShBjsLtGGiFHeEmXTInyaEom9ev7jHmW9rE3GmboHhGIDgC1O5jFWuykfY5P6YkP3XlVbaqjWVoyTE49aUYxFdcaptup+q3TQ3G80thhiLcCtJoc4aL+/2MrrL/uqTBNk7+BnO5kHl+RPB6GR7xBi8lxqlec4CPOpktEEqYLNnXpBwATpRdOeZxUCJkgVbO7UCwImSC+a9jyrEDBBqmBzp14QMEF60bTnWYWACVIFmzv1goAJ0oumPc8qBEyQKtjcqRcETJBeNO15ViFgglTB5k69IGCC9KJpz7MKAROkCjZ36gUBE6QXTXueVQiYIFWwuVMvCJggvWja86xCwASpgs2dekHABOlF055nFQImSBVs7tQLAiZIL5r2PKsQMEGqYHOnXhAwQXrRtOdZhYAJUgWbO/WCgAnSi6Y9zyoETJAq2NypFwRMkF407XlWIWCCVMHmTr0gYIL0omnPswoBE6QKNnfqBQETpBdNe55VCJggVbC5Uy8ImCC9aNrzrELABKmCzZ16QcAE6UXTnmcVAiZIFWzu1AsCJkgvmvY8qxAwQapgc6deEDBBetG051mFgAlSBZs79YKACdKLpj3PKgRMkCrY3KkXBEyQXjTteVYhYIJUweZOvSBggvSiac+zCgETpAo2d+oFAROkF017nlUImCBVsLlTLwiYIL1o2vOsQsAEqYLNnXpBwATpRdOeZxUCJkgVbO7UCwImSC+a9jyrEDBBqmBzp14QMEF60bTnWYWACVIFmzv1goAJ0oumPc8qBEyQKtjcqRcETJBeNO15ViFgglTB5k69IGCC9KJpz7MKAROkCjZ36gUBE6QXTXueVQiYIFWwuVMvCJggvWja86xCwASpgs2dekHABOlF055nFQImSBVs7tQLAiZIL5r2PKsQ+H/qKJnesbOAFwAAAABJRU5ErkJggg==", "language": "en-US", "id": "http://certificates.gamoeofthronesxyz.org/criteria/2016/08/got.json", "description": "This certifies that the named character is an official Game of Thrones character.", "title": "Game of Thrones Character"}}, "@context": "https://w3id.org/blockcerts/context#", "@type": "BlockchainCertificate", "receipt": {"proof": [{"left": "7dd64b40a24d5874e4024e0020456835125cd2040805b9da7b30af38a5ecfa16"}], "@type": "BlockchainReceipt", "merkleRoot": "40ab35359d9674907b32b323042d75de3ef3586bfd482b570749b15eb24a6d68", "anchors": [{"sourceId": "39119eca980f5110cc661428cd067053d2696d78b6f8350ab411acedb843610e", "type": "BTCOpReturn"}], "targetHash": "6502078d48cd734717e77fd672e7216e4cb5e8b21ddf1adf44bbb10e9750b21f", "type": "ChainpointSHA256v2"}}
```

Comment:
Same question here. It looks like the cert is modified, but the signature isn't, so this should be yet another tampered-with cert that fails validation, right?